apps/asap-server/test/routes/user.route.test.ts

```diff
@@ -95,6 +95,16 @@ describe('/users/ route', () => {
     });
 
     describe('Parameter validation', () => {
+      test('Should return a 400 error when additional properties exist', async () => {
+        const response = await supertest(appWithMockedAuth)
```

Comment:
maybe you could use test.each?

Same code as on line 293-301

Reply:
I understand the concern but in this case I'd prefer to be consistent with the existing code.
Let's discuss at a RTM if we should refactor authentication/validation tests and group them together.